lib/mrsk/commands/builder/multiarch/remote.rb

```diff
@@ -41,19 +41,11 @@ def create_context(arch, host)
 
     def remove_contexts
       combine \
-        remove_context(local["arch"]),
-        remove_context(remote["arch"])
+        remove_context(config.builder.local_arch),
+        remove_context(config.builder.remote_arch)
```

Comment:
Maybe setup a `delegate_to` for config.builder for remote/local_arch|host.

Reply:
Added delegation of all used public methods from `Mrsk::Configuration::Builder` to `builder_config`. Lines became shorter, readability improved.